styles.js

```diff
@@ -1,13 +1,89 @@
+import { css } from "styled-components";
 import { createGlobalStyle } from "styled-components";
 
-export default createGlobalStyle`
-  *,
-  *::before,
-  *::after {
+const globalStyles = css`
+  :root {
+    --max-width: 1100px;
+    --border-radius: 12px;
+    --foreground-rgb: 0, 0, 0;
+    --background-rgb: 214, 219, 220;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    :root {
+      --foreground-rgb: 255, 255, 255;
+      --background-rgb: 0, 0, 0;
+    }
+  }
+
+  * {
     box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+
+  html,
+  body {
+    max-width: 100vw;
+    overflow-x: hidden;
+    font-family: Roboto, Arial;
   }
+
   body {
-    margin: 0;
-    font-family: system-ui;
+    height: 100vh;
+    color: rgb(var(--foreground-rgb));
+    background: rgb(var(--background-rgb));
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    gap: 2rem;
+  }
+
+  a {
+    color: inherit;
+    text-decoration: none;
+  }
+
+  a:hover {
+    color: hotpink;
+    text-decoration: underline;
+  }
+
+  header {
+    padding: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+  }
+
+  footer {
+    width: 100%;
+    padding: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+  }
+
+  footer div {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-end;
+    gap: 1rem;
+  }
```

Comment:
If you have a footer and header component, these styles should probably be directly in the file or folder of the component

Reply:
i decide now for global styles and will refactor it maybe later

Reply:
@alinawinter 
It depends on your styling requirements.
This is the global meaning it's applied to every HTML tag.
You can see it as the default style. If your components need a unique look it's always possible to make them into styled-components.

@reyk-zepper you might remove the footer style if you don't need it.